proof/invariant-abstract/ARM_HYP/ArchVCPU_AI.thy

```diff
@@ -0,0 +1,729 @@
+(*
+ * Copyright 2022, ???
```

Comment:
Still need to add the proper copyright. The only thing I took from Sidney and Mitch's work was the definition of `active_cur_vcpu_of`. Should this be only UNSW then?

Reply:
Yes, that is good.